Build failure on powerpc64 and mips64

Thanks for the crate!  I'm trying to use it for Nix's CI tests.  However, the build fails for powerpc64-unknown-linux-gnu, powerpc64le-unknown-linux-gnu, mips64-unknown-linux-gnu, and mips64el-unknown-linux-gnu.
```
   Compiling caps v0.3.0
error[E0425]: cannot find value `CAPGET` in module `nr`
  --> /cargo/registry/src/github.com-1ecc6299db9ec823/caps-0.3.0/src/base.rs:12:40
   |
12 |     let r = unsafe { libc::syscall(nr::CAPGET, hdr, data) };
   |                                        ^^^^^^ not found in `nr`
error[E0425]: cannot find value `CAPSET` in module `nr`
  --> /cargo/registry/src/github.com-1ecc6299db9ec823/caps-0.3.0/src/base.rs:20:40
   |
20 |     let r = unsafe { libc::syscall(nr::CAPSET, hdr, data) };
   |                                        ^^^^^^ not found in `nr`
error: aborting due to 2 previous errors
error: Could not compile `caps`.
```

--- a/src/nr.rs
+++ b/src/nr.rs
@@ -69,8 +69,13 @@ pub const CAPGET: i64 = 90;
 pub const CAPSET: i64 = 91;
 
 #[cfg(target_arch = "powerpc")]
-pub const CAPGET: i64 = 183;
+pub const CAPGET: i32 = 183;
 #[cfg(target_arch = "powerpc")]
+pub const CAPSET: i32 = 184;
+
+#[cfg(target_arch = "powerpc64")]
+pub const CAPGET: i64 = 183;
+#[cfg(target_arch = "powerpc64")]
 pub const CAPSET: i64 = 184;
 
 #[cfg(target_arch = "mips")]
@@ -78,7 +83,17 @@ pub const CAPGET: i32 = 4204;
 #[cfg(target_arch = "mips")]
 pub const CAPSET: i32 = 4205;
 
+#[cfg(target_arch = "mips64")]
+pub const CAPGET: i64 = 5123;
+#[cfg(target_arch = "mips64")]
+pub const CAPSET: i64 = 5124;
+
 #[cfg(target_arch = "arm")]
 pub const CAPGET: i32 = 184;
 #[cfg(target_arch = "arm")]
 pub const CAPSET: i32 = 185;
+
+#[cfg(target_arch = "s390x")]
+pub const CAPGET: i64 = 184;
+#[cfg(target_arch = "s390x")]
+pub const CAPSET: i64 = 185;

